.github/workflows/.nvmrc

```
@@ -0,0 +1 @@
+echo "16.16.0" > .nvmrc
```

EOF

---

Also, 
```suggestion
16.16.0
```
And please put the file on the root of the project

---

@danielgalvis98 changes made